src/gen/lib/AST_grammar.ml

```diff
@@ -4,10 +4,26 @@
 
 type ident = string
 
+type alias = {
```

Comment:
A comment here would be useful, and an example.
For example with alias($._binary_minus, '-') what will be the value of id and name? (or maybe for those cases
there's no alias?)
And with alias($._literal_contents, $.bare_string)? what are id and name? (or maybe say that this is converted
into Symbol("_literal_contents", Some { id = ??; name = "bare_string")?